addons/website_sale_slides/models/product_product.py

```diff
@@ -8,3 +8,11 @@ class Product(models.Model):
     _inherit = "product.product"
 
     channel_ids = fields.One2many('slide.channel', 'product_id', string='Courses')
+
+    def get_product_multiline_description_sale(self):
+        payment_channels = self.channel_ids.filtered(lambda course: course.enroll == 'payment')
+
+        if not payment_channels:
+            return super(Product, self).get_product_multiline_description_sale()
+
+        return 'Access to:\n' + '\n'.join(payment_channels.mapped('name'))
```

Comment:
Translation marks ! And maybe use a better formating (like ``_('Access to %(formatted_channel_names)s', formatted_channel_name= ...)``).